fastlane/lib/fastlane/lane_manager.rb

```diff
@@ -103,18 +103,35 @@ def self.print_table(actions)
     # Lane chooser if user didn't provide a lane
     # @param platform: is probably nil, but user might have called `fastlane android`, and only wants to list those actions
     def self.choose_lane(ff, platform)
+      available = ff.runner.lanes[platform].to_a
+      if available.empty?
+        UI.error "It looks like you don't have any lanes to run just yet. Check out how to get started here: https://github.com/fastlane/fastlane 🚀"
+        exit
+      end
+
+      rows = []
+      available.each_with_index do |lane, index|
+        rows << [index + 1, lane.last.pretty_name, lane.last.description.first]
+      end
+
       loop do
-        UI.error "You must provide a lane to drive. Available lanes:"
-        available = ff.runner.available_lanes(platform)
+        UI.message "Welcome to fastlane! Here's what your app is setup to do:"
 
-        available.each_with_index do |lane, index|
-          UI.message "#{index + 1}) #{lane}"
-        end
+        rows << [rows.size + 1, "cancel", "No selection!"]
```

Comment:
that line needs to move outside of the loop (otherwise it will be added at each iteration)
and the table initialization can also move up (and doesn't need to be one at each loop).